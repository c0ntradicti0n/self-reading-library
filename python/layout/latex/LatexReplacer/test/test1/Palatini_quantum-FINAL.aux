\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\citation{1207.7214,1207.7235}
\citation{Englert:1964et,Higgs:1964ia,Higgs:1964pj}
\citation{1001.4538,1807.06211}
\citation{Starobinsky:1980te,Guth:1980zm,Linde:1981mu,Mukhanov:1981xt}
\citation{0710.3755}
\citation{0710.3755}
\citation{0803.2664}
\citation{1807.02376}
\citation{2001.10135}
\citation{Callan:1970ze}
\citation{0710.3755}
\citation{Palatini19,Einstein25}
\citation{0803.2664}
\citation{0803.2664}
\citation{1807.06211}
\citation{1303.3787}
\@writefile{toc}{\contentsline {section}{\numberline {1}Introduction}{1}{section.1}}
\newlabel{sec:introduction}{{1}{1}{Introduction}{section.1}{}}
\newlabel{action_full}{{1.1}{1}{Introduction}{equation.1.1}{}}
\citation{1008.5157}
\citation{0902.4465,0903.0355}
\citation{1012.2900}
\citation{1003.2635,1005.2978}
\citation{1003.2635}
\citation{1612.06253,1711.08761}
\citation{1008.5157}
\citation{1609.05209,1610.08916}
\citation{1008.5157,1412.3811}
\citation{1609.05209,1610.08916}
\citation{0812.3622,0812.4624}
\citation{1902.10148}
\citation{1010.1417,1307.5298,1501.02231,1701.07665}
\citation{1008.5157,Aydemir:2012nz}
\citation{1008.5157}
\citation{1412.3811}
\newlabel{cutoff}{{1.2}{2}{Introduction}{equation.1.2}{}}
\citation{1008.5157,1403.6078,1412.3811,Enckell:2016xse,1706.05007}
\citation{0904.1537}
\citation{0812.4950,0904.1537,1411.1923}
\citation{0809.2104,0812.4946,0904.1698,0910.1041}
\citation{1709.07853,1712.04874,1802.09299}
\citation{1012.2900}
\citation{0710.3755}
\citation{0803.2664}
\citation{0812.4624}
\@writefile{toc}{\contentsline {section}{\numberline {2}Metric vs Palatini: classical picture}{4}{section.2}}
\newlabel{sec:review}{{2}{4}{Metric vs Palatini: classical picture}{section.2}{}}
\newlabel{action}{{2.1}{4}{Metric vs Palatini: classical picture}{equation.2.1}{}}
\newlabel{FieldTransform1}{{2.2}{4}{Metric vs Palatini: classical picture}{equation.2.2}{}}
\newlabel{einsteinNonCanonical}{{2.3}{4}{Metric vs Palatini: classical picture}{equation.2.3}{}}
\newlabel{FieldTransform2}{{2.4}{4}{Metric vs Palatini: classical picture}{equation.2.4}{}}
\newlabel{einsteinAction}{{2.5}{4}{Metric vs Palatini: classical picture}{equation.2.5}{}}
\newlabel{Potential}{{2.6}{4}{Metric vs Palatini: classical picture}{equation.2.6}{}}
\citation{1807.06211}
\@writefile{lot}{\contentsline {table}{\numberline {1}{\ignorespaces Comparison between metric and Palatini Higgs inflation at tree level and for $\xi \gg 1$. In the Palatini case, we use $\xi = 10^7$. Since the analysis of this paper shows that $\xi $ can deviate from this value by an order of magnitude, we use the symbol ``$\approx $'' when displaying numerical values that depend on $\xi $. Expressions for $n_s$ and $r$ are given to the leading order in $N^{-1}$ and $\xi ^{-1}$. \relax }}{5}{table.caption.2}}
\providecommand*\caption@xref[2]{\@setref\relax\@undefined{#1}}
\newlabel{table}{{1}{5}{Comparison between metric and Palatini Higgs inflation at tree level and for $\xi \gg 1$. In the Palatini case, we use $\xi = 10^7$. Since the analysis of this paper shows that $\xi $ can deviate from this value by an order of magnitude, we use the symbol ``$\approx $'' when displaying numerical values that depend on $\xi $. Expressions for $n_s$ and $r$ are given to the leading order in $N^{-1}$ and $\xi ^{-1}$. \relax }{table.caption.2}{}}
\newlabel{slowRollParameters}{{2.8}{5}{Metric vs Palatini: classical picture}{equation.2.8}{}}
\newlabel{NofH}{{2.9}{5}{Metric vs Palatini: classical picture}{equation.2.9}{}}
\citation{1609.05209,1610.08916,1902.10148}
\citation{1906.11837}
\citation{1902.10148}
\citation{1807.06211}
\citation{1008.5157,1412.3811}
\citation{1008.5157,1412.3811}
\citation{tHooft:1972tcz}
\citation{1412.3811}
\newlabel{normalization}{{2.10}{6}{Metric vs Palatini: classical picture}{equation.2.10}{}}
\@writefile{toc}{\contentsline {section}{\numberline {3}Metric vs Palatini: quantum corrections}{6}{section.3}}
\newlabel{sec:newPhysics}{{3}{6}{Metric vs Palatini: quantum corrections}{section.3}{}}
\newlabel{mu}{{3.1}{6}{Metric vs Palatini: quantum corrections}{equation.3.1}{}}
\newlabel{Yt}{{3.2}{6}{Metric vs Palatini: quantum corrections}{equation.3.2}{}}
\citation{1412.3811}
\newlabel{counterTerms}{{3.5}{7}{Metric vs Palatini: quantum corrections}{equation.3.5}{}}
\newlabel{lambdaReplacement}{{3.7}{7}{Metric vs Palatini: quantum corrections}{equation.3.7}{}}
\citation{Bardeen:1995kv,0708.3550,1303.7244}
\@writefile{lot}{\contentsline {table}{\numberline {2}{\ignorespaces Comparison between the energy scales in metric and Palatini Higgs inflation.\relax }}{8}{table.caption.3}}
\newlabel{table2}{{2}{8}{Comparison between the energy scales in metric and Palatini Higgs inflation.\relax }{table.caption.3}{}}
\newlabel{deltaBetaMetric}{{3.8}{8}{Metric vs Palatini: quantum corrections}{equation.3.8}{}}
\newlabel{deltaLambdaMetric}{{3.9}{8}{Metric vs Palatini: quantum corrections}{equation.3.9}{}}
\citation{1308.2627,1402.1476}
\citation{1712.08471,1904.05699}
\citation{1712.04874}
\citation{1507.08833}
\citation{1205.2892}
\citation{1205.2893,1205.6497,1307.3536}
\citation{0904.1537}
\newlabel{deltaBetaPalatini}{{3.10}{9}{Metric vs Palatini: quantum corrections}{equation.3.10}{}}
\newlabel{deltaLambdaPalatini}{{3.11}{9}{Metric vs Palatini: quantum corrections}{equation.3.11}{}}
\@writefile{toc}{\contentsline {section}{\numberline {4}Connection of low- and high-energy physics}{9}{section.4}}
\newlabel{sec:connection}{{4}{9}{Connection of low- and high-energy physics}{section.4}{}}
\newlabel{xiOfLambda}{{4.1}{9}{Connection of low- and high-energy physics}{equation.4.1}{}}
\citation{Tanabashi:2018oca}
\citation{1709.07853,1802.09299}
\newlabel{runningCoupling}{{4.2}{10}{Connection of low- and high-energy physics}{equation.4.2}{}}
\newlabel{xiEquation}{{4.3}{10}{Connection of low- and high-energy physics}{equation.4.3}{}}
\newlabel{runningCondition}{{4.5}{10}{Connection of low- and high-energy physics}{equation.4.5}{}}
\newlabel{lambdaLowerBound}{{4.6}{10}{Connection of low- and high-energy physics}{equation.4.6}{}}
\newlabel{xiMinAnalytic}{{4.7}{10}{Connection of low- and high-energy physics}{equation.4.7}{}}
\citation{1709.07853}
\citation{1709.07853}
\citation{1509.04044}
\citation{1810.01772}
\citation{Tanabashi:2018oca}
\citation{1411.1923}
\citation{1412.3649,1801.03944}
\newlabel{xiMin}{{4.8}{11}{Connection of low- and high-energy physics}{equation.4.8}{}}
\newlabel{upperXi}{{4.9}{11}{Connection of low- and high-energy physics}{equation.4.9}{}}
\newlabel{topMassMax}{{4.10}{11}{Connection of low- and high-energy physics}{equation.4.10}{}}
\citation{0812.4950,0904.1537}
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces Non-minimal coupling $\xi $ as a function of the low-energy value of top Yukawa coupling $y_t^{\text  {low}}$, following from analytic (in blue) and numerical (in orange) analysis. Close to the lower bound on $\xi $, the analytic estimate predicts the solution that is absent in the numerical calculation. Otherwise, the analytic and numerical results agree with each other. Smaller values of $y_t^{\text  {low}}$ are also viable but not shown.\relax }}{12}{figure.caption.4}}
\newlabel{sfig:xiOfLambda0}{{1}{12}{Non-minimal coupling $\xi $ as a function of the low-energy value of top Yukawa coupling $y_t^{\text {low}}$, following from analytic (in blue) and numerical (in orange) analysis. Close to the lower bound on $\xi $, the analytic estimate predicts the solution that is absent in the numerical calculation. Otherwise, the analytic and numerical results agree with each other. Smaller values of $y_t^{\text {low}}$ are also viable but not shown.\relax }{figure.caption.4}{}}
\newlabel{topMassMaxResult}{{4.11}{12}{Connection of low- and high-energy physics}{equation.4.11}{}}
\citation{1008.5157,1403.6078,1412.3811}
\citation{0710.3755}
\citation{0803.2664}
\citation{1012.2900}
\citation{1008.5157}
\citation{1904.05237,1905.02302}
\citation{1904.05237}
\citation{1905.02302}
\newlabel{topMassMaxMetric}{{4.13}{13}{Connection of low- and high-energy physics}{equation.4.13}{}}
\@writefile{toc}{\contentsline {section}{\numberline {5}Discussion}{13}{section.5}}
\newlabel{sec:conclusion}{{5}{13}{Discussion}{section.5}{}}
\citation{Froggatt:1995rt}
\citation{0912.0208}
\citation{2001.09088}
\citation{2001.09088}
\citation{Asaka:2005an,Asaka:2005pn}
\citation{1005.3497,1010.1415,1103.5963}
\citation{mathematica}
\@writefile{toc}{\contentsline {section}{\numberline {A}Numerical lower bound on $\xi $}{14}{appendix.A}}
\newlabel{app}{{A}{14}{Numerical lower bound on $\xi $}{appendix.A}{}}
\bibstyle{JHEP}
\bibdata{inflationBib}
\bibcite{1207.7214}{{1}{}{{}}{{}}}
\bibcite{1207.7235}{{2}{}{{}}{{}}}
\bibcite{Englert:1964et}{{3}{}{{}}{{}}}
\bibcite{Higgs:1964ia}{{4}{}{{}}{{}}}
\bibcite{Higgs:1964pj}{{5}{}{{}}{{}}}
\bibcite{1001.4538}{{6}{}{{}}{{}}}
\bibcite{1807.06211}{{7}{}{{}}{{}}}
\bibcite{Starobinsky:1980te}{{8}{}{{}}{{}}}
\bibcite{Guth:1980zm}{{9}{}{{}}{{}}}
\bibcite{Linde:1981mu}{{10}{}{{}}{{}}}
\@writefile{lof}{\contentsline {figure}{\numberline {2}{\ignorespaces The slow-roll parameter $\epsilon $ as a function of the low-energy value of top Yukawa coupling $y_t^{\text  {low}}$, following from analytic (in blue) and numerical (in orange) analysis. The upper part of the curve corresponds to the lower part of Fig.\spacefactor \@m {}\xspace  \ref  {sfig:xiOfLambda0}.\relax }}{15}{figure.caption.6}}
\newlabel{sfig:epsilonOfLambda0}{{2}{15}{The slow-roll parameter $\epsilon $ as a function of the low-energy value of top Yukawa coupling $y_t^{\text {low}}$, following from analytic (in blue) and numerical (in orange) analysis. The upper part of the curve corresponds to the lower part of \fig \ref {sfig:xiOfLambda0}.\relax }{figure.caption.6}{}}
\bibcite{Mukhanov:1981xt}{{11}{}{{}}{{}}}
\bibcite{0710.3755}{{12}{}{{}}{{}}}
\bibcite{0803.2664}{{13}{}{{}}{{}}}
\bibcite{1807.02376}{{14}{}{{}}{{}}}
\bibcite{2001.10135}{{15}{}{{}}{{}}}
\bibcite{Callan:1970ze}{{16}{}{{}}{{}}}
\bibcite{Palatini19}{{17}{}{{}}{{}}}
\bibcite{Einstein25}{{18}{}{{}}{{}}}
\bibcite{1303.3787}{{19}{}{{}}{{}}}
\bibcite{1008.5157}{{20}{}{{}}{{}}}
\bibcite{0902.4465}{{21}{}{{}}{{}}}
\bibcite{0903.0355}{{22}{}{{}}{{}}}
\bibcite{1012.2900}{{23}{}{{}}{{}}}
\bibcite{1003.2635}{{24}{}{{}}{{}}}
\bibcite{1005.2978}{{25}{}{{}}{{}}}
\bibcite{1612.06253}{{26}{}{{}}{{}}}
\bibcite{1711.08761}{{27}{}{{}}{{}}}
\bibcite{1609.05209}{{28}{}{{}}{{}}}
\bibcite{1610.08916}{{29}{}{{}}{{}}}
\bibcite{1412.3811}{{30}{}{{}}{{}}}
\bibcite{0812.3622}{{31}{}{{}}{{}}}
\bibcite{0812.4624}{{32}{}{{}}{{}}}
\bibcite{1902.10148}{{33}{}{{}}{{}}}
\bibcite{1010.1417}{{34}{}{{}}{{}}}
\bibcite{1307.5298}{{35}{}{{}}{{}}}
\bibcite{1501.02231}{{36}{}{{}}{{}}}
\bibcite{1701.07665}{{37}{}{{}}{{}}}
\bibcite{Aydemir:2012nz}{{38}{}{{}}{{}}}
\bibcite{1403.6078}{{39}{}{{}}{{}}}
\bibcite{Enckell:2016xse}{{40}{}{{}}{{}}}
\bibcite{1706.05007}{{41}{}{{}}{{}}}
\bibcite{0904.1537}{{42}{}{{}}{{}}}
\bibcite{0812.4950}{{43}{}{{}}{{}}}
\bibcite{1411.1923}{{44}{}{{}}{{}}}
\bibcite{0809.2104}{{45}{}{{}}{{}}}
\bibcite{0812.4946}{{46}{}{{}}{{}}}
\bibcite{0904.1698}{{47}{}{{}}{{}}}
\bibcite{0910.1041}{{48}{}{{}}{{}}}
\bibcite{1709.07853}{{49}{}{{}}{{}}}
\bibcite{1712.04874}{{50}{}{{}}{{}}}
\bibcite{1802.09299}{{51}{}{{}}{{}}}
\bibcite{1906.11837}{{52}{}{{}}{{}}}
\bibcite{tHooft:1972tcz}{{53}{}{{}}{{}}}
\bibcite{Bardeen:1995kv}{{54}{}{{}}{{}}}
\bibcite{0708.3550}{{55}{}{{}}{{}}}
\bibcite{1303.7244}{{56}{}{{}}{{}}}
\bibcite{1308.2627}{{57}{}{{}}{{}}}
\bibcite{1402.1476}{{58}{}{{}}{{}}}
\bibcite{1712.08471}{{59}{}{{}}{{}}}
\bibcite{1904.05699}{{60}{}{{}}{{}}}
\bibcite{1507.08833}{{61}{}{{}}{{}}}
\bibcite{1205.2892}{{62}{}{{}}{{}}}
\bibcite{1205.2893}{{63}{}{{}}{{}}}
\bibcite{1205.6497}{{64}{}{{}}{{}}}
\bibcite{1307.3536}{{65}{}{{}}{{}}}
\bibcite{Tanabashi:2018oca}{{66}{}{{}}{{}}}
\bibcite{1509.04044}{{67}{}{{}}{{}}}
\bibcite{1810.01772}{{68}{}{{}}{{}}}
\bibcite{1412.3649}{{69}{}{{}}{{}}}
\bibcite{1801.03944}{{70}{}{{}}{{}}}
\bibcite{1904.05237}{{71}{}{{}}{{}}}
\bibcite{1905.02302}{{72}{}{{}}{{}}}
\bibcite{Froggatt:1995rt}{{73}{}{{}}{{}}}
\bibcite{0912.0208}{{74}{}{{}}{{}}}
\bibcite{2001.09088}{{75}{}{{}}{{}}}
\bibcite{Asaka:2005an}{{76}{}{{}}{{}}}
\bibcite{Asaka:2005pn}{{77}{}{{}}{{}}}
\bibcite{1005.3497}{{78}{}{{}}{{}}}
\bibcite{1010.1415}{{79}{}{{}}{{}}}
\bibcite{1103.5963}{{80}{}{{}}{{}}}
\bibcite{mathematica}{{81}{}{{}}{{}}}
\providecommand\NAT@force@numbers{}\NAT@force@numbers
