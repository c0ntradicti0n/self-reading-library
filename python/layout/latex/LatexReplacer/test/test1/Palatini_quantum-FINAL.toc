\contentsline {section}{\numberline {1}Introduction}{1}{section.1}
\contentsline {section}{\numberline {2}Metric vs Palatini: classical picture}{4}{section.2}
\contentsline {section}{\numberline {3}Metric vs Palatini: quantum corrections}{6}{section.3}
\contentsline {section}{\numberline {4}Connection of low- and high-energy physics}{9}{section.4}
\contentsline {section}{\numberline {5}Discussion}{13}{section.5}
\contentsline {section}{\numberline {A}Numerical lower bound on $\xi $}{14}{appendix.A}
