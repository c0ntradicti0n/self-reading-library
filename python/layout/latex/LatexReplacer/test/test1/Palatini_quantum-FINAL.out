\BOOKMARK [1][-]{section.1}{Introduction}{}% 1
\BOOKMARK [1][-]{section.2}{Metric vs Palatini: classical picture}{}% 2
\BOOKMARK [1][-]{section.3}{Metric vs Palatini: quantum corrections}{}% 3
\BOOKMARK [1][-]{section.4}{Connection of low- and high-energy physics}{}% 4
\BOOKMARK [1][-]{section.5}{Discussion}{}% 5
\BOOKMARK [1][-]{appendix.A}{Numerical lower bound on }{}% 6
